data/org.freedesktop.impl.portal.Notification.xml

```diff
@@ -47,6 +47,12 @@
 
         The format of the @notification is the same as for
         :ref:`org.freedesktop.portal.Notification.AddNotification`.
+        It additionally can contain the following keys:
+
+        * ``desktop-file-id`` (``s``)
+
+          The desktop file of the app adding the notification. It can be
```

Comment:
Best to mention that this id contains the `.desktop` prefix.

Reply:
Yup, makes sense.